to_do_list/src/Components/List.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
please call the file the same name you call the default exported class.
this way it's much easier to understand from the import what you're actually importing, and easier to understand what class you actually wanted to import.